File: server/src/main/java/org/hkurh/doky/dto/DocumentDto.java
package org.hkurh.doky.dto;

public class DocumentDto {

    private Long id;
    private String name;
    private String description;
    private UserDto creator;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public UserDto getCreator() {
        return creator;
    }

    public void setCreator(UserDto creator) {
        this.creator = creator;
    }
}

File: server/src/main/java/org/hkurh/doky/documents/DocumentDto.kt
package org.hkurh.doky.documents

import org.hkurh.doky.users.UserDto

class DocumentDto {
    var id: Long? = null
    var name: String? = null
    var description: String? = null
    var creator: UserDto? = null
    var createdBy: String? = null
    var modifiedBy: String? = null
}
